website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -32,6 +34,23 @@ if (process.env.DISCORD_CLIENT_ID) {
   );
 }
 
+if (boolean(process.env.NODE_ENV === "development")) {
```

Comment:
```suggestion
if (process.env.NODE_ENV === "development") {
```

We can remove the dependency now.